package.json

```diff
@@ -15,7 +15,8 @@
     "Anton Whalley <anton.whalley@nearform.com> (https://github.com/No9)",
     "Matteo Collina <matteo.collina@gmail.com> (https://github.com/mcollina)",
     "Pedro Teixeira <pedro.teixeira@gmail.com> (https://github.com/pgte)",
-    "James Halliday <mail@substack.net> (https://github.com/substack)"
+    "James Halliday <mail@substack.net> (https://github.com/substack)",
+    "Gordon Hall <gordonh@member.fsf.org> (https://github.com/bookchin)"
```

Comment:
Oh! I didn't mean to imply that by that addition - force of habit (I generally encourage any contributor to "sign my yearbook"), but given that our work on @Storj makes extensive use of this project, I'd be thrilled to continue to contribute!